src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,33 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private int maxDiggingDepth;
```

Comment:
Don't forget about access modifiers for your custom fields. It's good practice to make fields private and provide getters and setters to access them, which you have done correctly here. However, please ensure that all fields in similar classes also follow this rule.